lib/index.js

```diff
@@ -145,6 +152,18 @@ class GraphQLComponent {
   get mocks() {
     return this._mocks;
   }
+
+  get directives() {
+    return this._directives;
+  }
+
+  get importedDirectives() {
+    return this._importedDirectives;
+  }
+
+  get mergedDirectives() {
```

Comment:
Same question — public api?

Reply:
yes, this one will be needed to merge directive's schema with executable schema later in server/framework code